ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.Android.CSharp/MGNamespace.csproj

```diff
@@ -68,7 +73,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="MonoGame.Content.Builder" Version="3.7.0.9" />
-    <PackageReference Include="MonoGame.Framework.Android" Version="3.7.0.1708" />
+    <PackageReference Include="MonoGame.Framework.Android" Version="3.7.*" />

```

Comment:
Nuget reports warning when you use *, dunno why.

Reply:
Hmm, I don't get any.